src/XRP/wallet.ts

```diff
@@ -27,8 +22,13 @@ export interface WalletGenerationResult {
  * A wallet object that has an address and keypair.
  */
 class Wallet {
-  private readonly publicKey: string
-  private readonly privateKey: string
+  /**
+   * The default derivation path to use with BIP44.
+   */
+  public static defaultDerivationPath = "m/44'/144'/0'/0/0"
```

Comment:
Make this a static property on the class so I don't have to document `getDefaultDerivationPath()`.